pkg/logs/consumer/minio.go

```diff
@@ -0,0 +1,170 @@
+package consumer
+
+import (
```

Comment:
I would go with writing stream in chunks. As we can't implement writing on the end of stream. 

So each second logs should be flushed to the minio.

Reply:
Added saving on a 80 kB batch